homeassistant/components/ovo_energy/config_flow.py

```diff
@@ -0,0 +1,66 @@
+"""Config flow to configure the OVO Energy integration."""
+import logging
+
+import aiohttp
+from ovoenergy.ovoenergy import OVOEnergy
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from .const import CONF_ACCOUNT_ID, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@config_entries.HANDLERS.register(DOMAIN)
```

Comment:
The decorator is legacy. Remove that and pass the domain as domain keyword argument instead.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler#defining-your-config-flow